app/code/Magento/Cms/view/adminhtml/layout/cms_block_index.xml

```diff
@@ -9,6 +9,7 @@
     <body>
         <referenceContainer name="content">
             <uiComponent name="cms_block_listing"/>
+            <block class="Magento\Backend\Block\Template" template="Magento_Cms::url_filter_applier.phtml" />
```

Comment:
I think we can add `Urlfilter` component in ui_compmentn instead of creating a new block. Please check the similar implementation
https://github.com/magento/adobe-stock-integration/blob/f8f7257d30bf54646536491ad5366ae082c13570/AdobeStockImageAdminUi/view/adminhtml/ui_component/adobe_stock_images_listing.xml#L55-L70
Let me know your thoughts
Thanks

Reply:
@konarshankar07 I think both approaches are fine

Reply:
@sivaschenko ... Yes, you are right but my concerns is the namespace are hard coded in the `url_filter_applier.phtml` file which can be easily exported using ui_component.